.github/workflows/conda.yml

```
@@ -0,0 +1,56 @@
+name: Testing with Conda
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
```

```suggestion
      - uses: actions/checkout@v2
        with:
          fetch-depth: 0
```
This is needed otherwise conda cannot find the git tag to use as a version `1.1.0` in our case